configs/image_restorers/liif/metafile.yml

```diff
@@ -29,6 +30,7 @@ Models:
 - Config: configs/image_restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py
   In Collection: LIIF
   Metadata:
+    GPUs: "\u25B3"
```

Comment:
Parse error, I will solve it by updating ``.dev_scripts/github/update_model_index.py``